### PR TITLE
Modify S5756: LaYC format

### PR DESCRIPTION
**Pull request by guillaume-dequenne-sonarsource**
## Review

A dedicated reviewer checked the rule description successfully for:

- [ ] logical errors and incorrect information
- [ ] information gaps and missing content
- [ ] text style and tone
- [ ] PR summary and labels follow [the guidelines](https://github.com/SonarSource/rspec/#to-modify-an-existing-rule)

### CODE REVIEW

**Review by joke1196 — CHANGES_REQUESTED** [review]
Looks good, I have just added a small comment

File: rules/S5756/python/rule.adoc [hunk 1]
```diff
@@ -1,16 +1,22 @@
+This rule raises an issue when a call is made to a non-callable object.
+
 == Why is this an issue?
 
 :link-with-uscores1: https://docs.python.org/3/reference/datamodel.html#object.__call__
 
-In order to be callable, a python class should implement the {link-with-uscores1}[``++__call__++``] method.
+In order to be callable, a Python class should implement the {link-with-uscores1}[``++__call__++``] method.
```
[the remark this answers is not in the document]
Comment (guillaume-dequenne-sonarsource) [reply rules/S5756/python/rule.adoc]: I added a small sentence saying that and moved the code example to the first tab, hopefully that should help.

**Review by joke1196 — APPROVED** [review]
LGTM!

**Review by jean-jimbo-sonarsource — APPROVED** [review]
LGTM

**Pull request merged**